docs/system/system-metrics.md

```diff
@@ -575,14 +575,14 @@ This metric is [recommended][MetricRecommended].
 <!-- semconv metric.system.network.dropped(full) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| `network.io.direction` | string | The network IO operation direction | `send` | Recommended |
 | `system.device` | string | The device identifier | `(identifier)` | Recommended |
-| `system.network.direction` | string |  | `transmit` | Recommended |
 
-`system.network.direction` MUST be one of the following:
+`network.io.direction` MUST be one of the following:
 
 | Value  | Description |
 |---|---|
-| `transmit` | transmit |
+| `send` | send |
```

Comment:
_We discussed this briefly during Kubecon, but I'd like to delve into it a bit deeper here for broader attention._

"send" sounds better to me as well. I agree that "transmit" can potentially be interpreted as both "send" and "receive". But it's still unclear why transmit/receive (or tx/rx) are widely adopted in the industry. It'd be great to see if any other references are using "send"/"receive" instead. Did you have a chance to look for it? I'd like to make sure we are not making it more confusing for the users.

Reply:
When I was originally considering this in the Process Metrics PR, I thought `send` and `receive` made the most sense because of that verbiage already being commonly used in user-space programs thanks to that being the verbiage in libc's socket library.

It does seem that in hardware level, `transmit` is the more widely used term. This is likely because of its usage in hardware terminology, and as its part of the `Transmission Control Protocol` acronym.

So it seems like `transmit` when you think at a hardware level, `send` when you think at a software level.

Even outside of those terms, Elastic Common Schema use some alternates like `ingress` and `egress` (https://www.elastic.co/guide/en/ecs/current/ecs-network.html#field-network-direction), or various network metrics in GCP using terminology like `inbound` and `outbound`, or `SERVER_IN` and `SERVER_OUT` (https://cloud.google.com/monitoring/api/metrics_gcp).

I'm torn on this point, I still like send and receive but I don't feel as strongly about it as I did before. But another question is whether these semantic conventions have to pick two names; could we provide examples of different options and then at an implementation level (in the collector namely) choose the two that feel best?

Reply:
After thinking about it a bit more, I no longer think there's a compelling enough reason to switch from `transmit` to `send`. I still like `send` better, but I don't feel strongly enough to fight for it.

I'd like to revisit the thing I mentioned before where we could provide examples of what they would be and choose the implementation in the collector in the end, but I am not sure what the precedent is like for that elsewhere in semconv.

For now I'm going to resolve this conversation, and we could probably bring it back up another time or we can just keep `transmit` and not worry about it.

Reply:
Sounds good to me. Filing an issue with all the findings would be great. We can keep it as is, but we probably should make the decision clear. I think OTel semantic conventions should prescribe this since the values are rather enums than random strings.